Shared/Flows/Onboarding/OnboardingViewController.swift

```diff
@@ -70,37 +69,22 @@ class OnboardingViewController: SwitchableContentViewController<OnboardingConten
             try await self.updateInvitor(with: newWelcomeViewController.benjiId)
         }
 
-//        self.welcomeVC.$state.mainSink { (state) in
-//            switch state {
-//            case .welcome:
-//                self.updateUI()
-//            case .login:
-//                Task {
-//                    try await self.updateInvitor(with: "IQgIBSPHpE")
-//                }
-//                self.current = .phone(self.phoneVC)
-//            case .reservationInput:
-//                self.updateUI()
-//            case .foundReservation(let reservation):
-//                self.reservationId = reservation.objectId
-//                if let identity = reservation.createdBy?.objectId {
-//                    Task {
-//                        try await self.updateInvitor(with: identity)
-//                    }
-//                }
-//                self.current = .phone(self.phoneVC)
-//            case .reservationError:
-//                self.updateUI()
-//            }
-//        }.store(in: &self.cancellables)
+        self.welcomeVC.onDidComplete = { [unowned self] result in
+            switch result {
+            case .success:
+                self.current = .phone(self.phoneVC)
+            case .failure:
+                break
+            }
+        }
```

Comment:
We don't have a login button at the moment so just send the user to the phone entry.

Reply:
Can you just add one below the waitlist for now? I think we will need it for testing.

Reply:
@Benjamindodgson Yep, no problem.